src/editor/index.html

```diff
@@ -155,7 +156,6 @@
     <script src="libs/codemirror/addon/show-invisibles/show-invisibles.js"></script>
 
     <script src="init.js"></script>
```

Comment:
It's for semantic reasons. Loading and execution order of js code is *not* asynchronous, and "init.js" is meant to be loaded first :)